1_fetch/src/get_feature_vars.R

```diff
@@ -198,9 +268,9 @@ prep_feature_vars <- function(sb_var_data, sites_all, sites_screened,
                                  class == 7 ~ "BARREN", class == 8 ~ "FOREST", 
                                  class == 9 ~ "FOREST", class == 10 ~ "FOREST", 
                                  class == 11 ~ "GRASSLAND", class == 12 ~ "SHRUBLAND", 
-                                 class == 13 ~ "CROPLAND", class == 14 ~ "HAY/PASTURE", 
+                                 class == 13 ~ "CROPLAND", class == 14 ~ "HAYPASTURE", 
                                  class == 15 ~ "WETLAND", class == 16 ~ "WETLAND", 
-                                 class == 17 ~ "ICE/SNOW")) %>%
+                                 class == 17 ~ "ICESNOW")) %>%
```

Comment:
In [this issue](https://github.com/USGS-R/regional-hydrologic-forcings-ml/pull/163/files) we convert these from `/` to `-`.
```suggestion
                                 class == 17 ~ "ICE-SNOW")) %>%
```

Reply:
Oh, okay. I didn't know that would happen with character names